ConsoleApp/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace ConsoleApp
+{
+    using System;
+    class ConsoleApp
+    {
+        static void ShowValue(int value)
+        {
+            Console.Write("+" + (value < 0 ? "(" + value + ")" : value));
+        }
+        static long SumCalculate(int x, int y)
+        {
+            long sum = 0;
+            Console.Write("Sum = ");
+            if (x < y)
+            {
+                Console.Write("{0}", x);
+                sum = x;
+                do
+                {
+                    sum += ++x;
+                    ShowValue(x);
+                } while (x != y);
+            }
+            else if (y < x)
+            {
+                Console.Write("{0}", y);
+                sum = y;
+                do
+                {
+                    sum += ++y;
+                    ShowValue(y);
+                } while (x != y);
+            }
+            else
+            {
+                Console.Write("{0}", x);
+                sum = x;
+            }
+            return sum;
+        }
+        static int ReadConsoleValue(string str)
+        {
+            string? ConsoleString = null;
+            bool success = false;
+            int value = 0;
+
+            while (success == false)
```

Comment:
not critical, but it's better to replace with !success instead of success == false

Reply:
replaced condition to "!success"